pkg/shell/simple.html

```diff
@@ -42,7 +42,7 @@
               </li>
               <li class="divider"></li>
               <li>
-                <a id="go-logout" translatable="yes">Log out</a>
+                <a id="go-logout" translatable="yes">Log Out</a>
```

Comment:
This was already patched in the previous PR, so this shouldn't be here any more. But GitHub is happy, I guess git/github are clever enough to ignore bits which are already patched the same way.